build_debian.sh

```diff
@@ -188,7 +188,20 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     usbutils                \
     pciutils                \
     iptables-persistent     \
-    logrotate
+    logrotate               \
+    grub2-common
+
+# Python tabulate required for sonic-utilities
+python_tabulate_url=http://ftp.ca.debian.org/debian/pool/main/p/python-tabulate/python-tabulate_0.7.7-1_all.deb
```

Comment:
uploaded this package to here. can you use this url
http://packages.microsoft.com/repos/sonic-dev/pool/main/p/python-tabulate/python-tabulate_0.7.7-1_all.deb